src/Elastic-2.0.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license isOsiApproved="false" licenseId="Elastic-2.0" name="Elastic License 2.0" listVersionAdded="3.16">
+    <crossRefs>
+      <crossRef>https://www.elastic.co/licensing/elastic-license</crossRef>
+      <crossRef>https://github.com/elastic/elasticsearch/blob/master/licenses/ELASTIC-LICENSE-2.0.txt</crossRef>
+    </crossRefs>
+    <notes>This license was released on February 3, 2021</notes>
+    <text>
+      <titleText>
+        <p>Elastic License 2.0</p>
+      </titleText>
+      <p>URL: https://www.elastic.co/licensing/elastic-license</p>
+      <p>## Acceptance</p>
+      <p>By using the software, you agree to all of the terms and conditions below.</p>
+      <p>## Copyright License</p>
+      <p>
+        The licensor grants you a non-exclusive, royalty-free, worldwide,
+        non-sublicensable, non-transferable license to use, copy, distribute, make
+        available, and prepare derivative works of the software, in each case subject to
+        the limitations and conditions below.
+      </p>
+      <p>## Limitations</p>
+      <p>
+        You may not provide the software to third parties as a hosted or managed
+        service, where the service provides users with access to any substantial set of
+        the features or functionality of the software.
+      </p>
+      <p>
+        You may not move, change, disable, or circumvent the license key functionality
+        in the software, and you may not remove or obscure any functionality in the
+        software that is protected by the license key.
+      </p>
+      <p>
+        You may not alter, remove, or obscure any licensing, copyright, or other notices
+        of the licensor in the software. Any use of the licensor’s trademarks is subject
+        to applicable law.
+      </p>
+      <p>## Patents</p>
+      <p>
+        The licensor grants you a license, under any patent claims the licensor can
+        license, or becomes able to license, to make, have made, use, sell, offer for
+        sale, import and have imported the software, in each case subject to the
+        limitations and conditions in this license. This license does not cover any
+        patent claims that you cause to be infringed by modifications or additions to
+        the software. If you or your company make any written claim that the software
+        infringes or contributes to infringement of any patent, your patent license for
+        the software granted under these terms ends immediately. If your company makes
+        such a claim, your patent license ends immediately for work on behalf of your
+        company.
+      </p>
+      <p>## Notices</p>
+      <p>
+        You must ensure that anyone who gets a copy of any part of the software from you
+        also gets a copy of these terms.
+      </p>
```

Comment:
I'm not sure I'd agree, to be honest  :)  I guess I'm seeing these more like heading tags rather than bullets.

For similar MarkDown-formatted licenses, I think we have some that have called out the formatting characters as optional and some that haven't. E.g. [BlueOak-1.0.0](https://github.com/spdx/license-list-XML/blob/master/src/BlueOak-1.0.0.xml) tags the MarkDown characters as `<optional>`, while [PolyForm-Noncommercial-1.0.0](https://github.com/spdx/license-list-XML/blob/master/src/PolyForm-Noncommercial-1.0.0.xml) doesn't.

More generally, we might want to figure out a consistent approach for how we want to represent MarkDown-formatted licenses in the SPDX XML templates. But since we aren't currently consistent, I'm inclined to merge this one as-is for now. What do you think?

Reply:
Happy to change it or leave it as is. I even used PolyForm as a template (having seen it from kemitchell before I knew that it looked somewhat similar).

Reply:
As @goneall suggested below, I've filed #1395 for us to talk about this further. For this one, I think we can leave as-is. Thank you both!